src/main/resources/api.yaml

```diff
@@ -1136,6 +1188,42 @@ components:
           type: string
         otherProperties:
           type: string
+    CommentDto:
+      type: object
+      required:
+        - commentId
+        - ownerId
+        - rating
+        - title
+        - description
+      properties:
+        commentId:
+          type: string
+          format: uuid
+        ownerId:
+          type: string
+        rating:
+          type: integer
+          format: int16
+        title:
+          type: string
+        description:
+          type: string
+    CommentCreationDto:
```

Comment:
I think ownerId is not required as part of request body - this can be provided by UserIdentityService at the backend site.
Other thing that the `title` field seems to be missing here

Reply:
right, ownerId is not necessary, but title is not missing